customisation/back-translation.md

```diff
@@ -1,6 +1,18 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Back-translation
-description:
+description: Back-translating or back-copying target language sentences to augment parallel data
 ---
+
+**Back-translation** is the process of translating the monolingual data in the target language into the source language and then back into the target language.
+The goal is to generate synthetic [parallel data](/customisation/parallel-data.md) that can be used to train machine translation systems.
```

Comment:
This is a great place to introduce the "augment" term!
Perhaps we can say that synthetic data is necessary to augment parallel data so that there is more data to train machine translation systems and improve quality.

Reply:
"necessary" is not always true, we can be less opinionated.